### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
What is missing: 
(1) Registering this documentation for rendering and publishing at docs.typo3.org with the [webhook](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocForExtension/Webhook.html#github).
(2) Remove any documentation URL from the "External manual" field of the TER extension page configuration at https://extensions.typo3.org as the documentation at docs.typo3.org is detected automatically and a specific URL is error-prone.
(3) Insert composer package name "friendsoftypo3/taskcenter" into "Composer name of extension" field of the TER extension page configuration.
(4) Insert the repository URL "https://github.com/FriendsOfTYPO3/taskcenter" into "Link to repository" field of the TER extension page configuration.
(5) Insert the repository issues URL "https://github.com/FriendsOfTYPO3/taskcenter/issues" into "Link to issue tracker" field of the TER extension page configuration.

**Comment by alexander-nitsche** [comment]
How it looks:
Screen Shot 2022-04-05 at 19 00 54

**Comment by alexander-nitsche** [comment]
Added new commit regarding additional sources (TER, repository and docs.typo3.org) in composer.json.

Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/185

File: ext_emconf.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 $EM_CONF[$_EXTKEY] = [
     'title' => 'User>Task Center',
-    'description' => 'The Task Center is the framework for a host of other extensions.',
+    'description' => 'The Task Center provides the framework into which other extensions hook, for example, the sys_action extension.',
```
Comment (ohader) [inline ext_emconf.php:4]: Sounds vague, but I don't have a better description at hand right now...

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
TER foo done

**Pull request merged**